io_context_test.go

```diff
@@ -56,7 +56,9 @@ func TestIOContext(t *testing.T) {
 
 func TestOpenIOContext(t *testing.T) {
 	path := filepath.Join(t.TempDir(), "iocontext.txt")
-	c, err := OpenIOContext(path, NewIOContextFlags(IOContextFlagWrite))
+	d := NewDictionary()
```

Comment:
I don't really see the point of providing a real `Dictionary` here since we're not asserting anything linked to it below, therefore you can remove
```go
d := NewDictionary()
_ = d.Set("k", "v", 0)
```